kratos/solving_strategies/builder_and_solvers/builder_and_solver.h

```diff
@@ -590,6 +590,22 @@ class BuilderAndSolver
     {
     }
 
+    /**
+     * @brief It reconstructs the solution vector at slave DOFs
+     * @param pScheme The pointer to the integration scheme
+     * @param rModelPart The model part to compute
+     * @param rDx The vector of unkowns
+     */
+    virtual void ReconstructSlaveSolution(
+        typename TSchemeType::Pointer pScheme,
+        ModelPart& rModelPart,
+        TSystemMatrixType& rA,
+        TSystemVectorType& rDx,
+        TSystemVectorType& rb
+        )
+    {
```

Comment:
Should we add a warning or error?

Reply:
I think a warning could be useful in case this function is called from the base class

Reply:
added warning